app/views/example-template.html

```diff
@@ -84,17 +84,13 @@
 {% endblock %}
 
 {% block headerEnd %}
-  <div class="site-notifications">
-  </div>
+{% endblock %}
+
+{% block pageBreadcrumbs %}
```

Comment:
Do we need this on `app/views/example-template.html` as it seems to be inherited from `app/views/example.html` ?